lib/index.js

```diff
@@ -19,6 +19,7 @@ var defaults = {
   debug: identity,
   lint: true,
   minify: false,
+  encapsulateStyles: true,
```

Comment:
Do you think this should the be default? I would expect using something like this would require users to opt in.